src/bp/ui-studio/src/web/components/Util/form.util.ts

```diff
@@ -0,0 +1 @@
+export const isMissingCurlyBraceClosure = text => (text?.match(/{/g) || []).length !== (text?.match(/}/g) || []).length
```

Comment:
Can you make the logic of this piece a bit clearer? Ex: typings, and ensuring that text is indeed a string (in case we pass an object with languages as keys)